pom.xml

```diff
@@ -203,8 +203,9 @@
                     <version>${maven.release.plugin.version}</version>
                     <configuration>
                         <autoVersionSubmodules>true</autoVersionSubmodules>
-                        <preparationGoals>clean verify install</preparationGoals>
+                        <preparationGoals>clean install</preparationGoals>
                         <pushChanges>false</pushChanges>
+                        <arguments>-Prelease ${arguments}</arguments>
```

Comment:
Yes. Doesn't even look like we have a way to generate and aggregate them right now.